lms/static/sass/base/_reset.scss

```diff
@@ -96,4 +96,5 @@ td { vertical-align: top; }
   @page { margin: 0.5cm; }
   p, h2, h3 { orphans: 3; widows: 3; }
   h2, h3 { page-break-after: avoid; }
+  .donotprint { display:none !important; }
```

Comment:
Sometimes, it is useful to have a method of making sure that something in your course does not print (for example, if it only really has meaning in an interactive webpage). This defines a class called "donotprint" that makes sure that that element does not print, but shows normally on the screen.